src/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  padding: 50;
+}
+
+.stars--5 .stars__star {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:not(:nth-child(n+3)) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars__star:not(:nth-child(n+4)) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:not(:nth-child(n+5)) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
These rules are the same for all selectors, so they could be written with commas and shorten the code.